cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -440,6 +440,20 @@ declare namespace Deno {
    */
   export function osRelease(): string;
 
+  /**
+   * Returns the system uptime in number of seconds.
+   *
+   * ```ts
+   * console.log(Deno.uptime());
+   * ```
+   *
+   * Requires `allow-sys` permission.
+   *
+   * @tags allow-sys
+   * @category Runtime Environment
+   */
+  export function uptime(): number;
```

Comment:
Little bike-shedding maybe it should be `Deno.osUptime()`?

Reply:
`Deno.osUptime()` is more clear and falls in line with the naming convention of `Deno.osRelease()`. I definitely prefer it over `Deno.uptime()`.

I would also like to bikeshed another alternative being `Deno.systemUptime()` (which is in line with `Deno.systemMemoryInfo()`. I don't have strong feelings either way.

Reply:
After checking few other places, I decided to go with `osUptime`, as:

> the uptime is a kernel internal value, which ticks up every cycle, it starts counting when the kernel has initialized. That is, when the first cycle has ended. Even before anything is mounted, directly after the bootloader gives control to the kernel image.

which means that IMO it should be treated as `Operating System` info, where `System` info should be more "hardware-like" related,  as in mem/cpu/peripherals etc.